ffi/derive/tests/ui_fail/non_robust_repr_c.stderr

```diff
@@ -0,0 +1,19 @@
+error[E0277]: the trait bound `bool: ReprC` is not satisfied
+ --> tests/ui_fail/non_robust_repr_c.rs:4:38
+  |
+4 | #[derive(Clone, Copy, PartialEq, Eq, FfiType)]
+  |                                      ^^^^^^^ the trait `ReprC` is not implemented for `bool`
```

Comment:
If you look at docs for `ReprC` you will see it requires for the type to be **robust**, not only `repr(C)`

Reply:
This library doesn't allow types that have trap representations to cross ffi boundary without checking that bit-value is indeed valid for the given type. If we allowed this it could cause UB. In case of the `bool` type, it only has two valid bit-values in Rust, but all bit-values are valid in C